### PR TITLE
update ja/docs/_index.md

### PR DESCRIPTION
**Pull request by movingfinger**
from #25203

### CODE REVIEW

File: content/ja/docs/_index.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 ---
+linktitle: Kubernetes ドキュメント
```
Comment (inductor) [inline content/ja/docs/_index.md:2]:
```suggestion
linktitle: Kubernetesドキュメント
```

**Review by inductor — CHANGES_REQUESTED** [review]

**Comment by inductor** [comment]
@movingfinger 
Thank you for your contribution. CLA check returns error. Please your fix your commit with your GitHub information.

File: content/ja/docs/_index.md (hunk 1 above)
Comment (movingfinger) [reply content/ja/docs/_index.md]: コメントありがとうございます

**Pull request closed without merging**